Tests/PSKoans.tests.ps1

```diff
@@ -0,0 +1,120 @@
+$here = (Split-Path -Parent $MyInvocation.MyCommand.Path) -replace 'Tests', ''
+$sut = "PSKoans.psm1"
+Import-Module "$here\$sut" -Prefix "TST"
```

Comment:
Is the fact that `Initialize-KoanDirectory -FirstImport` effectively called when loading the module a concern here? Just trying to avoid having invoking tests create a koans directory / potentially overwrite it just for the sake of running tests.

It's not a huge concern now, but as we get more and more koan files... it might become more important later on.

Reply:
Agreed, I like the idea of individual module files in the repo. Then the build process combines them into a single root module.

I will try to get around to that tonight.